File: python/test/prime_factorize.test.py
# verification-helper: PROBLEM https://judge.yosupo.jp/problem/factorize


from python.lib.prime_factrize import prime_factorize as pf


q = int(input())
for _ in range(q):
  ai = int(input())
  r = pf(ai)
  out = ""
  for pi,ei in r:
    out += (str(pi)+" ")*ei
  print(len(r),out)

File: cpp/test/prime_factorize.test.cpp
#define PROBLEM http://judge.u-aizu.ac.jp/onlinejudge/description.jsp?id=NTL_1_A

#include <iostream>
#include "cpp/lib/prime_factorize.hpp"
using namespace std;


int main(){
    int n; cin >> n;
    auto r = prime_factorize(n);
    cout << n << ":";
    for (auto pe:r){
        for (int i=0; i<pe.second;++i){
            cout << " " << pe.first;
        }
    }
    cout << endl;
}